spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/properties/KafkaConsumerProperties.java

```diff
@@ -85,7 +85,8 @@ public long getReferencePoint() {
 	}
 
 	/**
-	 * When autoCommitOffset is true, this setting dictates whether to commit the offset after each record is processed.
+	 * When autoCommitOffset is true, this setting dictates whether to commit the offset
+	 * after the complete batch of records returned by a poll() is processed.
```

Comment:
This doesn't reflect the change I suggested.

Perhaps - when `true` the offset is committed after each record, otherwise the offsets for the complete set of records received from the poll() are committed after all records have been processed.